### PR TITLE
Modify S6327: Improve the recommended fix

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6327/recommended.adoc [hunk 1]
```diff
@@ -1,3 +1,8 @@
 == Recommended Secure Coding Practices
 
-It's recommended to encrypt SNS topics that contain sensitive information. Encryption and decryption are handled transparently by SNS, so no further modifications to the application are necessary.
+It is recommended to encrypt SNS topics that contain sensitive information.
+
+To do so, create a master key and affect the SNS topic to it. Without a master
+key, called `KmsMasterKeyId` in diverse SDKs, the SNS topic is not encrypted by deafult.
```
Comment (loris-s-sonarsource) [inline rules/S6327/recommended.adoc:6]:
```suggestion
key, the SNS topic is not encrypted by default.
```

File: rules/S6327/recommended.adoc [hunk 2]
```diff
@@ -1,3 +1,13 @@
 == Recommended Secure Coding Practices
 
-It's recommended to encrypt SNS topics that contain sensitive information. Encryption and decryption are handled transparently by SNS, so no further modifications to the application are necessary.
+It is recommended to encrypt SNS topics that contain sensitive information.
+
+To do so, create a master key and affect the SNS topic to it. Without a master
```
Comment (hendrik-buchwald-sonarsource) [inline rules/S6327/recommended.adoc:5]: Is "affect" the right word here?

File: rules/S6327/recommended.adoc [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+key, the SNS topic is not encrypted by default.
```
Comment (hendrik-buchwald-sonarsource) [inline rules/S6327/recommended.adoc:6]: Using the negative version is usually a bit harder to understand than the positive version: "With a master key, the SNS topic is encrypted by default".

**Review by hendrik-buchwald-sonarsource — CHANGES_REQUESTED** [review]
I would recommend a little bit of rewording.

File: rules/S6327/recommended.adoc (hunk 3 above)
Comment (loris-s-sonarsource) [reply rules/S6327/recommended.adoc]: I removed it, the goal was to say that by default no encryption applied but I am starting to think this sentence, at this location could be confusing. I replaced it with notable information instead

**Review by hendrik-buchwald-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**